src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,22 @@
-export const TodoList = () => {};
+type Props = {
+  title: string;
+  setTitle: (s: string) => void;
+  errorTitle: boolean;
+};
+export const TodoList: React.FC<Props> = ({ title, setTitle, errorTitle }) => {
+  return (
+    <div className="field">
+      <input
+        type="text"
+        data-cy="titleInput"
+        value={title}
+        placeholder="Enter new todo "
+        onChange={(e) => setTitle(e.target.value)}
+      />
+
+      {errorTitle && title === '' ? (
```

Comment:
```suggestion
      {errorTitle ? (
```
better to reset error state for this input in onChange handler